lib/payoneer/payout.rb

```diff
@@ -2,14 +2,18 @@ module Payoneer
   class Payout
     CREATE_PAYOUT_API_METHOD_NAME = 'PerformPayoutPayment'
 
-    def self.create(program_id:, payment_id:, payee_id:, amount:, description:, payment_date: Time.now, currency: 'USD')
+    def self.create(program_id:, payment_id:, payee_id:, amount:, description:,
+                    payment_date: Time.current, currency: 'USD')
+      unless payment_date.is_a? String
+        payment_date = payment_date.in_time_zone('EST').strftime('%m/%d/%Y %H:%M:%S')
+      end
```

Comment:
Good catch on that. It is weird that Payoneer would take EST, specifically. I wonder if the docs mean to say that they expect US Eastern time (EST or EDT). @troq would you be able to send them an email for verification on this? Is it supposed to be EST or do they acknowledge daylight saving time? Or are they just tricking us and they're really expecting UTC?